src/components/calcite-loader/calcite-loader.stories.ts

```diff
@@ -22,17 +22,17 @@ export const Simple = (): string => html`
   />
 `;
 
-export const NoPadding = (): string => html`
-  <div style="border: 1px solid rgb(192,192,192, 0.5); width: 100px">
-    <calcite-loader
-      active
-      type="${select("type", ["determinate", "indeterminate"], "indeterminate")}"
-      scale="${select("scale", ["s", "m", "l"], "m")}"
-      ${boolean("no-padding", true)}
-      value="${number("value", 0, { range: true, min: 0, max: 100, step: 1 })}"
-    />
-  </div>
-`;
+// export const NoPadding = (): string => html`
```

Comment:
Can we just remove instead of commenting out?

Reply:
We're just going to ignore this region in Screener that way we don't have to remove anything.